demo/Example+SLRealizer+Usage.ipynb

```diff
@@ -0,0 +1,251 @@
+    "\n",
+    "We will use the same observation catalog in our realization of lenses and non-lenses. Called `minion_1016`, this is the current official Reference Simulated Survey for the LSST. Let's begin by defining the path to our observation catalog."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is the directory where all our data files will be stored.\n",
+    "data_dir = os.path.join(os.environ['SLREALIZERDIR'], 'data')\n",
+    "# Observation catalog filename\n",
+    "observation_fname = os.path.join(data_dir, 'twinkles_observation_history.csv')\n",
+    "# Observation catalog dataframe\n",
+    "observation_df = pd.read_csv(observation_fname).query(\"(filter != 'y')\").reset_index(drop=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note I have queried out entries with the y-band; this is simply because the SDSS does not have a y-band!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Realizing lensed quasar systems with OM10Realizer"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We first define the paths for the output source and object tables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output_lens_source_path = os.path.join(data_dir, 'lens_source_table.csv')\n",
+    "output_lens_object_path = os.path.join(data_dir, 'lens_object_table.csv')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then use OM10's native interface instead of Pandas to read in the OM10 catalog. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from om10 import DB\n",
+    "\n",
+    "# OM10 catalog filename\n",
+    "om10_fname = os.path.join(data_dir, 'qso_mock.fits')\n",
+    "# OM10's DB object\n",
+    "om10_db = DB(catalog=om10_fname)\n",
+    "# Selection to simulate LSST conditions\n",
+    "om10_db.select_random(maglim=23.3, area=1.e2, IQ=0.75)\n",
+    "# Giving colors using a framework called LensPop (Collett 2015)\n",
+    "om10_db.paint(synthetic=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "More about the selection line:  `maglim` is the 10-sigma point source detection limit and `IQ` is the survey image quality. `area` refers to the total survey area, but entering a number above 1.e5 sq. ft. allows us to get all the lenses that meet our `maglim` and `IQ` criteria. We advise keeping the area small, say 100.0, if you just want to see that the cell runs!\n",
```

Comment:
The OM10 `maglim` refers to the 3rd brightest quasar image in the lens system - this is important, because it's _not_ what we care about most. When querying a blended object catalog, it's the total magnitude (all QSO images and the lens galaxy) that we can select on. To start with, we need to be inclusive by setting maglim to be faint - and then applying a cut in total magnitude later.

Reply:
I think for this notebook we should just explain this, and keep moving. In some applications it _will_ be appropriate to select on m3.

Reply:
Done